api/v1alpha1/capability_properties.go

```diff
@@ -0,0 +1,76 @@
+package v1alpha1
```

Comment:
Please try to generate the csv.
I realised types which are not in the same file as the main types file does not get added. Therefore we moved all together into dynakube_types

Reply:
moved CapabilityProperties type into dynakube_types